### ISSUE TITLE
“Service Category” section and sub section “Restoration” and “Custom” are non-responsive

### ISSUE THREAD

**Issue by fixmykix** [fixmykix/android-latest#9]
These Pages are Non-Responsive, and I dont remember them ever being apart of the design.

**Comment by ankitgoyaltech** [comment]
Please share the screenshot to be more precise

**Comment by fixmykix** [comment]
Yes I will add screenshots am working on this. I do not have android and am managing this feedback so it has been a it difficult for android.
